_posts/2015-11-18-Stats-2015.markdown

```diff
@@ -0,0 +1,123 @@
+
+The QIs provide a positive feedback loop, showing you ways in which you can game the system by improving your library. By focusing on showing you ways to improve rather than penalising discouraged behaviour I hope we can continue to foster a welcome community.
+
+To expand on that further and allow people to have a better sense of their the state of their libraries, we created Pod Author Pages. Here's a great example from [Andrea Mazzini](https://cocoapods.org/owners/734) making it really easy to understand his impact on the community.
+
+[pic]
+
+Another multi-year project that we shipped this year was [CocoaPods Stats](/Stats/). Stats offers a way of knowing how many downloads a pod has, and far more interestingly the number of targets it has been integrated with. This is something I haven't found in other dependency managers, as it's a really hard problem to solve. We couldn't have shipped this alone, so a big thanks to [Segment](https://segment.com) for sponsoring a lot of the infrastructure here.
+
+While the number of pods available via CocoaPods has increased in the last year, the amount of servers we need for search has not. [Florian Hanke](https://github.com/floere) who has been working with us since the start on search continues to refuse to throw more money at our search server. His work has made it possible for us to deal with doubling our search traffic, support more metadata, add different sort types and allow scoping to all the new platforms Apple keep shipping.
+
+[CocoaDocs](http://cocoadocs.org), the CocoaPods documentation service started supporting Swift code this year via Jazzy, and the future is looking bright on that front as Jazzy now supports Objective-C projects and one of these days will support mixed language projects. We'll probably be migrating CocoaDocs to only use Jazzy next year. CocoaDocs had problems scaling with the number of new pod versions being shipped this year. We remedied this by working with [Button](https://www.usebutton.com) to get a high-end Mac Mini doing the work instead of a [shared VPS](/CocoaDocs-Colo/).
+
+Finally, we made it easier to get bootstrapped to work on the web infrastructure via a meta-repo called [Strata](https://github.com/CocoaPods/Strata).
+
+So, to wrap this up, I was looking at the Google Analytics for the CocoaPods web properties. My favourite odd statistic is this: We get more user sessions from Windows devices than we do iOS. Perhaps this mobile thing is just a fad?
+
+## CocoaPods
+
+So this year was a crazy year from the perspective of Apple releases. 
+
+We got to a point where in the last 3 months, Swift accounts for half of the new Pods. At this point last year we were just starting to ship betas supporting frameworks, enabling Swift for those brave enough to be early adopters. Now there's a Swift library is in the top ten downloaded Pods and there are tens of thousands of apps using CocoaPods with Swift frameworks.
```

Comment:
Could be worth linking to Alamofire or Quick or another early adopter:

> those brave enough to be early adopters

Or maybe we could find the _first_ Swift pod? Nah, it's probably some random thing. Maybe find the oldest one above a certain QI?